packages/playground/src/dashboard/components/public_ips_table.vue

```diff
@@ -163,19 +169,24 @@ export default {
       }
       loadingIps.value = false;
     }
-    async function removeFarmIp(options: RemoveFarmIPModel, index: number) {
+    async function removeFarmIp() {
       try {
         isRemoving.value = true;
-        await gridStore.grid.farms.removeFarmIp({ ip: options.ip, farmId: options.farmId });
-        createCustomToast("IP is deleted successfully!", ToastType.success);
-        publicIps.value.splice(index, 1);
+        for (const item of itemsToDelete.value) {
+          await gridStore.grid.farms.removeFarmIp({
+            ip: item.ip,
+            farmId: props.farmId,
+          });
+          createCustomToast("IP is deleted successfully!", ToastType.success);
+          await getFarmByID(props.farmId);
+        }
```

Comment:
When considering a list of more than 20 IP addresses, displaying them all in a toast notification might overwhelm the user. Instead, it's better to consolidate the information and display only one concise message after removing all the IPs. What are your thoughts on this approach?

Reply:
totally agree